com.ibm.wala.cast.js/source/com/ibm/wala/cast/js/loader/JavaScriptLoader.java

```diff
@@ -263,7 +262,7 @@ public SSAPutInstruction PutInstruction(int iindex, int ref, int value, String f
             byte[] utf8 = field.getBytes("UTF-8");
             return PutInstruction(iindex, ref, value, 
                 FieldReference.findOrCreate(JavaScriptTypes.Root, Atom.findOrCreate(utf8, 0, utf8.length), JavaScriptTypes.Root));
-          } catch (UnsupportedEncodingException e) {
+          } catch (@SuppressWarnings("unused") UnsupportedEncodingException e) {
```

Comment:
These are weird.  I think there are plenty of legitimate cases where you catch an exception and the handling does not make use of the exception object.  Do we really need to suppress these?  Can we shut off the Eclipse warning instead?